resources/assets/sass/app.scss

```diff
@@ -110,3 +111,22 @@ img {
     background-color: $btn-default-bg;
     color: $brand-success;
 }
+
+.landing p {
+    font-size: 2em;
+    margin-bottom: 1em;
+
+    &:last-child {
+        margin-bottom: 0;
```

Comment:
Could drop this last child and then adjust the margin top on the screen shot?

Also we should probably pull screen shot styles in here too